dash/frontend/src/app/modules/private/pages/cluster/gate-keeper/gate-keeper.component.html

```diff
@@ -39,11 +39,11 @@ <h2 class="text-style-green" *ngIf="gatekeeperStatusLoaded && gatekeeperInstalla
 
               <div class="col-xs-12 col-sm-12 col-md-2 col-lg-2">
                 <div class="text-align-right">
-                  <button  mat-raised-button color="primary" (click)="openInstallWizard()" *ngIf="gatekeeperConstraintTemplates === null; else recommendationButton">
+                  <button mat-raised-button color="primary" [allowedRoles]="['ADMIN', 'SUPER_ADMIN']" (click)="openInstallWizard()" *ngIf="gatekeeperConstraintTemplates === null; else recommendationButton">
```

Comment:
Were you able to test that this doesn't show up when for READ_ONLY users?

Reply:
Yes,
unsure if your question is more aimed at the allowedRoles or the gatekeeper setup.

I ran against my local docker desktop cluster, so I was able to clear my gatekeeper and check this page before & after installing.

The allowedRoles decorator basically adds 'display: none' to its parent element when the currently signed in user does not have one of the specified roles

Reply:
The allowedRoles :)
Did you sign in as the READ_ONLY user and make sure? I had previously noticed that the decorator wasn't always 100% effective.

Reply:
Yes, the allowed Roles always works as long as it is in the correct element. I think I know exactly what circumstance you are thinknig it doesn't work in, because I fixed a spot where it looked like it wasn't working, and Git said you added it. I'll comment on that spot and explain.